ocean_lib/assets/test/test_utils.py

```diff
@@ -100,3 +129,38 @@ def test_remove_trusted_algorithm(publisher_ocean_instance):
 
     assert new_publisher_trusted_algorithms is not None
     assert len(new_publisher_trusted_algorithms) < len(publisher_trusted_algorithms)
+
+
+def test_remove_unexisting_trusted_algorithm(publisher_ocean_instance):
```

Comment:
can you please combine the tests? It seems we have the same publishing setup on these tests. And the setup with publishing a ddo is costly. If we publish DDOs for each test case, that is more likely an integration test and not a unit test.

Possible suggestions: You can use a sample DDO (with compute service) instead. OR you can sequentially run the trusted_algorithm tests on the same published dataset.

Reply:
I think I will go with the first option (to use DDO sample) since I started to create separate functions.